packages/ui/src/views/landing-page.tsx

```diff
@@ -71,7 +72,16 @@ export const LandingPageView: React.FC<LandingPageProps> = ({ spaces, useTransla
                 )}
               </DropdownMenuContent>
             </DropdownMenu>
-          </div>
+            <Button
```

Comment:
should we center this button and have it below select project dropdown not have this side by side with  select project dropdown?

Reply:
Lets take this up later once the page has been reviewed by the design team? All of this is just temporary to complete the flow.